docker-compose.connect.yml

```diff
@@ -1,27 +1,33 @@
 version: '2.1'
 services:
-  #
-  # mender-deviceconnect
-  #
-  mender-deviceconnect:
-    image: mendersoftware/deviceconnect:mender-master
-    command: server --automigrate
-    extends:
-      file: common.yml
-      service: mender-base
-    networks:
-      - mender
-    depends_on:
-      - mender-mongo
-    environment:
-      DEVICECONNECT_MONGO_URL: "mongodb://mender-mongo"
-      DEVICECONNECT_NATS_URL: "nats://mender-nats"
 
-  mender-nats:
-    image: nats:2.1-alpine3.11
-    networks:
-      - mender
+    #
+    # mender-deviceconnect
+    #
+    mender-deviceconnect:
+        image: mendersoftware/deviceconnect:mender-master
+        command: server --automigrate
+        extends:
+            file: common.yml
+            service: mender-base
+        networks:
+            - mender
+        depends_on:
+            - mender-mongo
+        environment:
+            DEVICECONNECT_MONGO_URL: "mongodb://mender-mongo"
+            DEVICECONNECT_NATS_URL: "nats://mender-nats"
 
-  mender-workflows-worker:
-      environment:
-          HAVE_DEVICECONNECT: 1
+    mender-nats:
+        image: nats:2.1-alpine3.11
+        networks:
+            - mender
+
+    mender-workflows-worker:
+        environment:
+            DEVICECONNECT_ADDR: "mender-deviceconnect:8080"
```

Comment:
consider adding `http://` ?

Reply:
All workflows have the http:// hard-coded, see for example:
https://github.com/mendersoftware/workflows/blob/master/worker/update_device_inventory.json#L11

We can change it if we think it is useful, but not in this PR because it is out of scope.

Reply:
right. thanks. during the review I noticed that workflows differs in that from the rest, where we pass http:// in the env was for URL. right. perhaps a low priority follow up on this one.